recipes/r-spdl/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = '0.0.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-spdl
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/spdl_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/spdl/spdl_{{ version }}.tar.gz
+  sha256: a78da42e40d8a96041f72eedb427defd865134527c835610c7e24526ba0e3ca2
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcppspdlog >=0.0.10
+  run:
+    - r-base
+    - r-rcppspdlog >=0.0.10
+
+test:
+  commands:
+    - $R -e "library('spdl')"           # [not win]
+    - "\"%R%\" -e \"library('spdl')\""  # [win]
+
+about:
+  home: https://github.com/eddelbuettel/spdl
+  license: GPL-2.0-or-later
+  summary: Logging functions in 'RcppSpdlog' provide access to the logging functionality from
+    the 'spdlog' 'C++' library. This package offers shorter convenience wrappers for
+    the 'R' functions which match the 'C++' functions, namely via, say, 'spdl::debug()'
+    at the debug level. The actual formatting is done by the 'fmt::format()' function
+    from the 'fmtlib' library (that is also 'std::format()' in 'C++20' or later).
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - shelnutt2
+    - eddelbuettel
```

Comment:
@shelnutt2 the recipe looks good. Adding Dirk Eddelbuettel here requires his consent regarding becoming conda recipe maintainer of the package.

@eddelbuettel Are you fine with / interested in being conda-forge recipe maintainer of your cran spdl package?

Reply:
@Shelnutt2 Can you please remove Dirk from the list of recipe co-maintainers? You can also add me if you like

Reply:
Sorry I first the 'formal consent' part and potential blockers. 

I am happy to be one of several maintainers for this.